src/gettingstarted/dev-environments/create-environment.md

```diff
@@ -0,0 +1,52 @@
+# Development Environments
+
+## Create environment
+
+Platform.sh supports the deployment of isolated development environments from your branches, complete with exact copies of all of your data. This is useful for testing changes in isolation before merging them.
+
+
+<asciinema-player src="/videos/asciinema/branch.cast" preload=1 autoplay=1 loop=1></asciinema-player>
```

Comment:
We should be able to hand-edit the video to skip some of the waiting time.  Also, ssh key validation again.